crates/rspack_plugin_css/src/dependency/url.rs

```diff
@@ -102,7 +104,44 @@ impl DependencyTemplate for CssUrlDependency {
         .module_graph_module_by_dependency_id(self.id())
       && let Some(target_url) = self.get_target_url(&mgm.module_identifier, compilation)
     {
-      source.replace(self.start, self.end, format!("url({target_url})").as_str(), None);
+      let content = format!("url({})", css_escape_string(&target_url));
+      source.replace(self.start, self.end, &content, None);
     }
   }
 }
+
+static WHITE_OR_BRACKET_REGEX: Lazy<Regex> =
+  Lazy::new(|| Regex::new(r#"[\n\t ()'"\\]"#).expect("Invalid Regexp"));
+static QUOTATION_REGEX: Lazy<Regex> =
+  Lazy::new(|| Regex::new(r#"[\n"\\]"#).expect("Invalid Regexp"));
+static APOSTROPHE_REGEX: Lazy<Regex> =
+  Lazy::new(|| Regex::new(r#"[\n'\\]"#).expect("Invalid Regexp"));
+
+fn css_escape_string(s: &str) -> String {
```

Comment:
Where does this logic come from?

Reply:
https://github.com/webpack/webpack/blob/6be4065ade1e252c1d8dcba4af0f43e32af1bdc1/lib/dependencies/CssUrlDependency.js#L81